herd/parseTest.ml

```diff
@@ -51,6 +51,12 @@ module Top (TopConf:RunTest.Config) = struct
       let precision = TestConf.precision
       let variant = TestConf.variant
     end in
+    if Conf.variant Variant.Ifetch then begin
```

Comment:
Could we move this check to the cat file? Like an additional flag?

I think something like this in cat would work:
```
flag ~empty (if "vmsa" && "memtag" then ob else 0) as combining-vmsa-and-memtag-is-not-supported
```
it's a bit of a hack but it does the job.

or an indirect check:
```
flag ~empty (tr-ib | tc-ib) as combining-vmsa-and-memtag-is-not-supported
```

In any case, I think we should also warn if one combines vmsa and metag

Reply:
Ah, thank you, that's a brilliant suggestion -- I agree that it's nicer to have this sort of a check in cat. Perhaps, the right course of action is to revert the relevant commit and add this to the cat-file PR.

(I like the "hack-y" way; perhaps, with an insignificant edit:
```
flag ~empty (if "vmsa" && "memtag" then _ else 0) as combining-vmsa-and-memtag-is-not-supported
```
)